src/main/java/ludum/dare/components/KeyboardControlComponent.java

```diff
@@ -12,14 +12,22 @@
  */
 public class KeyboardControlComponent extends InputComponent {
 
-    GDXControls keyboard;
+    private GDXControls keyboard;
+
+    private Set<InputAction> previousPresses = new HashSet<>();
+
+    private boolean inControl = true;
 
-    Set<InputAction> previousPresses = new HashSet<>();
 
     public KeyboardControlComponent() {
         keyboard = GDXControls.defaultMapping;
     }
 
+
+    public void inControl(boolean inControl) {
+        this.inControl = inControl;
+    }
```

Comment:
#nit Could put this in InputComponent so we don't have multiple implementations and can just rely on parent logic.